### PR TITLE
Release 0.44.3

### PR DESCRIPTION
**Pull request by qinqon**
**What this PR does / why we need it**:
Release 0.44.3

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: manifests/cluster-network-addons/0.44.3/cluster-network-addons-operator.0.44.3.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+                  - name: OVS_CNI_IMAGE
+                    value: quay.io/kubevirt/ovs-cni-plugin@sha256:d43d34ed4b1bd0b107c2049d21e33f9f870c36e5bf6dc1d80ab567271735c8da
+                  - name: OVS_MARKER_IMAGE
+                    value: quay.io/kubevirt/ovs-cni-marker@sha256:8ec72d6a3d58cea27a2b9a52e6ef4ee0d753b3ba989d24555bdfbf01a995a9c1
+                  - name: KUBEMACPOOL_IMAGE
+                    value: quay.io/kubevirt/kubemacpool@sha256:e2318880e4b71d7734cc30d070ec8a898934e408fc878287858b6ecb0d61fc20
+                  - name: MACVTAP_CNI_IMAGE
+                    value: quay.io/kubevirt/macvtap-cni@sha256:f20d5e56f8b8c1ab7e5a64e536b66f65aa688b2d1dc0b37e3c26c2af2b481266
+                  - name: OPERATOR_IMAGE
+                    value: quay.io/kubevirt/cluster-network-addons-operator:v0.44.3
```
Comment (oshoval) [inline manifests/cluster-network-addons/0.44.3/cluster-network-addons-operator.0.44.3.clusterserviceversion.yaml:181]: maybe worth to use anchor all over for 0.44.3 ?
and then no need to repeat it
https://support.atlassian.com/bitbucket-cloud/docs/yaml-anchors/

**Comment by RamLavi** [comment]
/retest

tests failed after 0 seconds.. looks like prow hiccup

**Comment by RamLavi** [comment]
the test fails on my local as well

**Comment by qinqon** [comment]
/retest
```
+ mkdir -p /tmp/deploy.bridge-marker.3Fu8/github.com/kubevirt/bridge-marker
+ git clone https://github.com/kubevirt/bridge-marker /tmp/deploy.bridge-marker.3Fu8/github.com/kubevirt/bridge-marker
Cloning into '/tmp/deploy.bridge-marker.3Fu8/github.com/kubevirt/bridge-marker'...
remote: Not Found
fatal: repository 'https://github.com/kubevirt/bridge-marker/' not found
+ rm -rf /tmp/deploy.bridge-marker.3Fu8
```

**Comment by qinqon** [comment]
We are going to reconstruct this release including https://github.com/kubevirt/cluster-network-addons-operator/pull/760

**Pull request closed without merging**